### PR TITLE
File Manager  - Fix unit tests & update TS definitions

### PR DESCRIPTION
**Pull request by maxerokh**

### CODE REVIEW

**Review by DokaRus — DISMISSED** [review]

**Review by DokaRus — COMMENTED** [review]
image

**Review by DokaRus — APPROVED** [review]

File: js/ui/file_manager/ui.file_manager.js [hunk 1]
```diff
@@ -364,9 +364,9 @@ class FileManager extends Widget {
             },
 
             /**
-            * @name dxTreeViewOptions.customThumbnail
+            * @name dxFileManagerOptions.customThumbnail
```
Comment (San4es) [inline js/ui/file_manager/ui.file_manager.js:367]: customThumbnail -> customizeThumbnail?
/cc @RomanTsukanov @TikhonovaTatyana

File: js/ui/file_manager/ui.file_manager.js [hunk 2]
```diff
@@ -364,9 +364,9 @@ class FileManager extends Widget {
...
             * @type function
-            * @type_function_param1 item:dxFileManagerItem
+            * @type_function_param1 fileItem:object
```
Comment (San4es) [inline js/ui/file_manager/ui.file_manager.js:369]: Has fileItem some interface? File type? Smth else?

File: js/ui/file_manager/ui.file_manager.js (hunk 1 above)
Comment (RomanTsukanov) [reply js/ui/file_manager/ui.file_manager.js:367]: We gotta discuss the names of all the API members

Comment (maxerokh) [reply js/ui/file_manager/ui.file_manager.js:367]: Of course, we'll prepare full list of public API and discuss with tech writers. At now, we constantly change public API therefore better choice is discuss it later.

**Pull request merged**